File: 프로그래머스/lv1/12931. 자릿수 더하기/자릿수 더하기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    int a;
    for(int i=100000000; i>=1; i=i/10){
        
        a = n/i;
        n = n - (a*i);
        answer = answer + a;
    }
    return answer;
}

File: 프로그래머스/lv0/120906. 자릿수 더하기/자릿수 더하기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        String s = Integer.toString(n);
        String[] strarr = s.split("");
        for(int i =0; i<strarr.length; i++){
            answer = answer + Integer.parseInt(strarr[i]);
        }
        return answer;
    }
}